-------------Error at 01/27/2019 09:48:43------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Server shutdown in progress
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:43------------- 

-------------Error at 01/27/2019 09:48:45------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:45------------- 

-------------Error at 01/27/2019 09:48:52------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot ping SQL connection : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteNonQuery()
   en Stump.ORM.Database.Execute(String sql, Object[] args) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 553
   en Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.BaseServer\ServerBase.cs:línea 461 
-------------Error at 01/27/2019 09:48:52------------- 

-------------Warn at 01/27/2019 09:48:52------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Try to Re-open the connection 
-------------Warn at 01/27/2019 09:48:52------------- 

-------------Error at 01/27/2019 09:48:53------------- 
 Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive -> 
	Cannot reopen the SQL connection : MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
   en MySql.Data.MySqlClient.NativeDriver.Open()
   en MySql.Data.MySqlClient.Driver.Open()
   en MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   en MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   en MySql.Data.MySqlClient.MySqlPool.GetConnection()
   en MySql.Data.MySqlClient.MySqlConnection.Open()
   en Stump.ORM.Database.OpenSharedConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 243
   en Stump.ORM.DatabaseAccessor.OpenConnection(Database database) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 106
   en Stump.ORM.DatabaseAccessor.OpenConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 140
   en Stump.Server.BaseServer.ServerBase.KeepSQLConnectionAlive() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.BaseServer\ServerBase.cs:línea 470 
-------------Error at 01/27/2019 09:48:53------------- 

-------------Error at 01/27/2019 09:48:53------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:53------------- 

-------------Error at 01/27/2019 09:48:53------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:53------------- 

-------------Error at 01/27/2019 09:48:54------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:54------------- 

-------------Error at 01/27/2019 09:48:54------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:48:54------------- 

-------------Error at 01/27/2019 09:49:02------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:02------------- 

-------------Error at 01/27/2019 09:49:06------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:06------------- 

-------------Error at 01/27/2019 09:49:07------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:07------------- 

-------------Error at 01/27/2019 09:49:08------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:08------------- 

-------------Error at 01/27/2019 09:49:12------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   en MySql.Data.MySqlClient.ExceptionInterceptor.Throw(Exception exception)
   en MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   en MySql.Data.MySqlClient.MySqlCommand.CheckState()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:12------------- 

-------------Error at 01/27/2019 09:49:13------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : System.InvalidOperationException: Connection must be valid and open.
   en MySql.Data.MySqlClient.ExceptionInterceptor.Throw(Exception exception)
   en MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   en MySql.Data.MySqlClient.MySqlCommand.CheckState()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:13------------- 

-------------Error at 01/27/2019 09:49:15------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:15------------- 

-------------Error at 01/27/2019 09:49:16------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:16------------- 

-------------Error at 01/27/2019 09:49:18------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:18------------- 

-------------Error at 01/27/2019 09:49:20------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:20------------- 

-------------Error at 01/27/2019 09:49:29------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:29------------- 

-------------Error at 01/27/2019 09:49:35------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:35------------- 

-------------Error at 01/27/2019 09:49:37------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:37------------- 

-------------Error at 01/27/2019 09:49:38------------- 
 Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback -> 
	Exception raised when processing TimerEntry Stump.Core.Timers.TimedTimerEntry (Callback = Stump.Server.AuthServer.Managers.AccountManager.Void RefreshBans(), Delay = 60000) in Stump.Server.BaseServer.Benchmark.BenchmarkedTaskPool : MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered during command execution. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Fatal error encountered attempting to read the resultset. ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Reading from the stream has failed. ---> System.IO.IOException: No se puede escribir datos de en la conexión de transporte: Se ha anulado una conexión establecida por el software en su equipo host. ---> System.Net.Sockets.SocketException: Se ha anulado una conexión establecida por el software en su equipo host
   en System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   en System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en MySql.Data.Common.MyNetworkStream.HandleOrRethrowException(Exception e)
   en MySql.Data.Common.MyNetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.TimedStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   en System.IO.BufferedStream.Read(Byte[] array, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.ReadFully(Stream stream, Byte[] buffer, Int32 offset, Int32 count)
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.LoadPacket()
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   en MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader()
   en Stump.ORM.Database.<Query>d__71`1.MoveNext() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 783
   en System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   en System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   en Stump.Server.AuthServer.Managers.AccountManager.RefreshBans() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\AccountManager.cs:línea 96
   en Stump.Core.Timers.TimedTimerEntry.Trigger() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Timers\TimedTimerEntry.cs:línea 128
   en Stump.Core.Threading.SelfRunningTaskPool.ProcessCallback(Object state) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.Core\Threading\SelfRunningTaskPool.cs:línea 223. 
-------------Error at 01/27/2019 09:49:38------------- 

-------------Fatal at 01/27/2019 10:28:05------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Connection must be valid and open.
 Source: MySql.Data -> Void Throw(System.Exception)
 Stack Trace:
   en MySql.Data.MySqlClient.ExceptionInterceptor.Throw(Exception exception)
   en MySql.Data.MySqlClient.MySqlConnection.Throw(Exception ex)
   en MySql.Data.MySqlClient.MySqlCommand.CheckState()
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   en MySql.Data.MySqlClient.MySqlCommand.ExecuteNonQuery()
   en Stump.ORM.Database.Update(String tableName, String primaryKeyName, Object poco, Object primaryKeyValue, IEnumerable`1 columns) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 1457
   en Stump.ORM.Database.Update(Object poco, Object primaryKeyValue, IEnumerable`1 columns) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 1488
   en Stump.ORM.Database.Update(Object poco) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 1478
   en Stump.Server.AuthServer.Managers.WorldServerManager.RemoveWorld(WorldServer world) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\Managers\WorldServerManager.cs:línea 267
   en Stump.Server.AuthServer.IPC.IPCClient.Disconnect() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\IPC\IPCClient.cs:línea 363
   en Stump.Server.AuthServer.IPC.IPCClient.ProcessReceive(Object sender, SocketAsyncEventArgs e) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\IPC\IPCClient.cs:línea 172
   en System.Net.Sockets.SocketAsyncEventArgs.OnCompleted(SocketAsyncEventArgs e)
   en System.Net.Sockets.SocketAsyncEventArgs.ExecutionCallback(Object ignored)
   en System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   en System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   en System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   en System.Net.Sockets.SocketAsyncEventArgs.FinishOperationAsyncFailure(SocketError socketError, Int32 bytesTransferred, SocketFlags flags)
   en System.Net.Sockets.SocketAsyncEventArgs.CompletionPortCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* nativeOverlapped)
   en System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pOVERLAP) 
-------------Fatal at 01/27/2019 10:28:05------------- 

-------------Error at 01/27/2019 10:53:58------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/27/2019 10:53:58------------- 

-------------Fatal at 01/27/2019 10:53:58------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Authentication to host '127.0.0.1' for user 'root' using method 'mysql_native_password' failed with message: Unknown database 'ilyaelle_auth'
 Source: MySql.Data -> Void AuthenticationFailed(System.Exception)
 Stack Trace:
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   en MySql.Data.MySqlClient.NativeDriver.Authenticate(String authMethod, Boolean reset)
   en MySql.Data.MySqlClient.NativeDriver.Open()
   en MySql.Data.MySqlClient.Driver.Open()
   en MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   en MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   en MySql.Data.MySqlClient.MySqlPool.GetConnection()
   en MySql.Data.MySqlClient.MySqlConnection.Open()
   en Stump.ORM.Database.OpenSharedConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 243
   en Stump.ORM.DatabaseAccessor.OpenConnection(Database database) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 106
   en Stump.ORM.DatabaseAccessor.OpenConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 140
   en Stump.Server.AuthServer.AuthServer.Initialize() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\AuthServer.cs:línea 109 
-------------Fatal at 01/27/2019 10:53:58------------- 

-------------Fatal at 01/27/2019 10:53:58------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Unknown database 'ilyaelle_auth'
 Source: MySql.Data -> MySql.Data.MySqlClient.MySqlPacket ReadPacket()
 Stack Trace:
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket() 
-------------Fatal at 01/27/2019 10:53:58------------- 

-------------Error at 01/27/2019 10:54:16------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/27/2019 10:54:16------------- 

-------------Fatal at 01/27/2019 10:54:17------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Authentication to host '127.0.0.1' for user 'root' using method 'mysql_native_password' failed with message: Unknown database 'ilyaelle_auth'
 Source: MySql.Data -> Void AuthenticationFailed(System.Exception)
 Stack Trace:
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   en MySql.Data.MySqlClient.NativeDriver.Authenticate(String authMethod, Boolean reset)
   en MySql.Data.MySqlClient.NativeDriver.Open()
   en MySql.Data.MySqlClient.Driver.Open()
   en MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   en MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   en MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   en MySql.Data.MySqlClient.MySqlPool.GetConnection()
   en MySql.Data.MySqlClient.MySqlConnection.Open()
   en Stump.ORM.Database.OpenSharedConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\PetaPoco.cs:línea 243
   en Stump.ORM.DatabaseAccessor.OpenConnection(Database database) en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 106
   en Stump.ORM.DatabaseAccessor.OpenConnection() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale\Core\Stump.ORM\DatabaseAccessor.cs:línea 140
   en Stump.Server.AuthServer.AuthServer.Initialize() en G:\Documentos ssd\MEGAsync Downloads\Emu nexeption\Émulateur opale - copia\server\Stump.Server.AuthServer\AuthServer.cs:línea 109 
-------------Fatal at 01/27/2019 10:54:17------------- 

-------------Fatal at 01/27/2019 10:54:17------------- 
 Stump.Server.BaseServer.ServerBase.HandleCrashException -> 
	 Crash Exception : Unknown database 'ilyaelle_auth'
 Source: MySql.Data -> MySql.Data.MySqlClient.MySqlPacket ReadPacket()
 Stack Trace:
   en MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   en MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket() 
-------------Fatal at 01/27/2019 10:54:17------------- 

-------------Error at 01/27/2019 10:54:52------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/27/2019 10:54:52------------- 

-------------Error at 01/27/2019 10:57:50------------- 
 Stump.Server.BaseServer.Plugins.PluginManager.LoadAllPlugins -> 
	Cannot load unexistant plugin path ./plugins/ 
-------------Error at 01/27/2019 10:57:50------------- 

-------------Error at 01/27/2019 15:02:45------------- 
 Stump.Server.AuthServer.Handlers.Connection.ConnectionHandler+<>c__DisplayClass6_0.<HandleClearIdentificationMessage>b__1 -> 
	Error while joining last used world server, connection aborted 
-------------Error at 01/27/2019 15:02:45------------- 

-------------Error at 01/27/2019 15:02:45------------- 
 Stump.Server.AuthServer.Handlers.Connection.ConnectionHandler+<>c__DisplayClass6_0.<HandleClearIdentificationMessage>b__1 -> 
	Error while joining last used world server, connection aborted 
-------------Error at 01/27/2019 15:02:45------------- 

